Rakefile

```diff
@@ -12,4 +12,12 @@ task :job_properties do
       end
     end
   end
+end
+
+desc "Update the ttar script used for cert management to the latest version available on GitHub"
+task :update_ttar do
+  require "open-uri"
+  File.open("src/ttar/ttar", "w") do |f|
+    IO.copy_stream(open("https://raw.githubusercontent.com/jhunt/ttar/master/ttar"), f)
+  end
```

Comment:
I wonder if jhunt would consider making a github release of ttar

Reply:
I bet he would.
https://github.com/jhunt/ttar/releases/tag/v1.0.0